NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# #4 Users can search for products to buy

# Description

A user should see a search bar when authenticated in the nav bar. The user should be able to search by product.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Checkout to ap-search-products branch
Run the app.
Log in as Admina 
Search for a kite or whellbarrow
yay it should work!


# Checklist:
